Parsers/ASimAuditEvent/Tests/Microsoft_Windows_AuditEvent_DataTest.csv

```diff
@@ -0,0 +1,12 @@
+﻿Result
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 1541 records (78.54%) for field [EventType] of type [Enumerated]: [""Stop""] (Schema:AuditEvent)"
+"(0) Error: 4 invalid value(s) (up to 10 listed) in 1960 records (99.9%) for field [ObjectType] of type [Enumerated]: [""Event Log"",""Scheduled Task"",""Service"",""Directory Service Object""] (Schema:AuditEvent)"
+"(2) Info: Empty value in 1592 records (81.14%)  in recommended field [Object] (Schema:AuditEvent)"
```

Comment:
Is there a specific reason why Object is going empty. According to the schema this value mandatory.

Reply:
I modified the parser after the tests